clusterloader2/pkg/prometheus/experimental.go

```diff
@@ -37,7 +39,7 @@ type prometheusDiskMetadata struct {
 
 const (
 	gcloudRetryInterval  = 20 * time.Second
-	snapshotRetryTimeout = 10 * time.Minute
+	snapshotRetryTimeout = 5 * time.Minute
```

Comment:
Why?

Reply:
After kubetest sends sigint, there is 15 minutes before it will terminate everything. Creating a snapshot takes ~30 seconds. I guess that if we do not succeed in 5 minutes it's ok to move on to remaining phases - like gathering logs.

Reply:
Isn't CL2 killed at this point already?

This change is unrelated to this PR - so let's not mix it here.

Reply:
This change is related.

When this PR will enter, the 15 minutes timeout after sigint will be shared between snapshot and logs. The more time we have for a snapshot, the less time we have for getting logs.

When kubetest will send sigint to current process, it will have 15 minutes timeout before sending sigkill and blocking additional processes. See https://github.com/kubernetes/test-infra/blob/master/kubetest/process/process.go#L200 for logic on Control instance.

The same global control object: https://github.com/kubernetes/test-infra/blob/c6e29bb385e0a93113937f08b61fbe1b9de5ee5c/kubetest/main.go#L52
Wrapper for executing test: https://github.com/kubernetes/test-infra/blob/c6e29bb385e0a93113937f08b61fbe1b9de5ee5c/kubetest/e2e.go#L224
Wrapper for getting logs: https://github.com/kubernetes/test-infra/blob/c6e29bb385e0a93113937f08b61fbe1b9de5ee5c/kubetest/gke.go#L478